src/solution.c

```diff
@@ -602,8 +599,15 @@ static void solution_thread(void *arg)
         ephemeris_unlock();
       }
 
-      /* Send the observations. */
```

Comment:
This confusing due to the way the silly `(u32)t_check` thing is used in firmware from before this PR.

The default settings for Piksi is that `soln_freq` is 10Hz, and `obs_output_divisor` is 2 to give effective rate of 5Hz. Thus you will get a position solution in SBP every 10Hz, and a observation message every 5Hz.

Due to some rearranging of the solution thread code in this PR, the 5Hz check was also covering the position solution code.

So now the time propagation (which is needed for the position solution), and the solution code are run at 5Hz, but the code that sends to observations only runs at 5Hz, and only when the receiver clock has been correctly set (`TIME_FINE`)